overlord/snapshotstate/backend/backend.go

```diff
@@ -476,9 +476,69 @@ func addDirToZip(ctx context.Context, snapshot *client.Snapshot, w *zip.Writer,
 
 var ErrCannotCancel = errors.New("cannot cancel: import already finished")
 
+// XXX: move to a new errutil ?
+//
+// MultiError is a helper to handle mutliple errors.
+type MultiError struct {
```

Comment:
as an exported struct type this doesn't make a lot of sense to me. What kind of error it is, what classes of error is it about?

if we want something reusable it probably would need to be a type wrapping just errs []error part, ready to use for actual error structs that are specific to classes of errors

Reply:
to be clear, the idea to share code for this is not bad, but doing this generically is a bit of a pain, that's why we likely haven't tried so far

Reply:
I unexported this now.